ci/packet/packet-cluster.lokocfg.envsubst

```diff
@@ -23,7 +23,13 @@ cluster "packet" {
 
   worker_pool "pool-1" {
     count     = 2
-    node_type = "c2.medium.x86"
+    node_type = "n2.xlarge.x86"
+
+    os_arch = "amd64"
+    os_channel = "stable"
+    os_version = "2345.3.1"
+    ipxe_script_url = "https://stable.release.flatcar-linux.net/amd64-usr/2345.3.1/flatcar_production_packet.ipxe"
```

Comment:
Can’t this be configured such that it picks up latest stable, always?

Reply:
Yes, just use `current` as version.
This is just a test PR to see when the issues with the `n2`s is resolved. When we test we want a consistent environment where we use the same version for the test even if it changed on Packet or by releasing a new Stable version which could introduce new bugs.

Reply:
Ah right, gotcha!